### PR TITLE
Braden Update clear command to delete expired or completed tasks

### LINKED ISSUE #140: As a user, I can delete all my expired/completed tasks at once
so that I don't have to delete them one by one.

### PR DESCRIPTION
**Pull request by BradenTeo**
- Updated Clear Command
- Updated UG/DG for Clear Command
- Added JUnit tests for Clear Command
- Edited Gradle to exclude 2 tests because they have errors that I'm currently unable to fix

Fixes #140

### CODE REVIEW

File: src/main/java/seedu/sherpass/command/AddCommand.java [hunk 1]
```diff
@@ -17,10 +17,12 @@ public class AddCommand extends Command {
...
-            + "A task contains a task description and an optional dates "
-            + "to finish the task by.\n\nTo execute the command,\nenter 'add <task_description> "
-            + "/by <task_due_date> /do_on <date_to_work_on_task>'.\n"
-            + "E.g. add do assignment /by 3/6/2022 /do_on 1/6/2022.\n\n"
+            + "A task contains a task description and optional dates to finish the task by.\n\n"
+            + "To execute the command,\nenter 'add <task_description> "
+            + "/by <task_due_date> /do_on <date_to_work_on_task>'.\n\n"
+            + "Examples:\n\tadd do assignment /by 3/6/2022 /do_on 1/6/2022\n"
+            + "\tadd project /by 5/6/2022\n\tadd attend lecture /do_on 23/6/2022\n"
+            + "\tadd survive uni\n\n"
```
Comment (jltha) [inline src/main/java/seedu/sherpass/command/AddCommand.java]: Can you help to move all the command constants into the constant package? Maybe can create a new class within that package to contain them. Thank you!

File: src/main/java/seedu/sherpass/task/Task.java [hunk 2]
```diff
@@ -193,4 +193,4 @@ public Task copy() {
         return new Task(this.identifier, this.description, this.byDateTime,
                 this.doOnStartDateTime, this.doOnEndDateTime);
     }
-}
+}
```
Comment (jltha) [inline src/main/java/seedu/sherpass/task/Task.java:196]: add newline

File: src/main/java/seedu/sherpass/command/DeleteCommand.java [hunk 3]
```diff
@@ -48,4 +48,4 @@ public void execute(TaskList taskList, Ui ui, Storage storage) {
             ui.showError(ERROR_INVALID_INDEX_MESSAGE);
         }
     }
-}
+}
```
Comment (jltha) [inline src/main/java/seedu/sherpass/command/DeleteCommand.java:51]: add newline to here

File: src/main/java/seedu/sherpass/command/AddCommand.java [hunk 4]
```diff
@@ -77,4 +77,4 @@ public void execute(TaskList taskList, Ui ui, Storage storage) {
             ui.showError(exception.getMessage());
         }
     }
-}
+}
```
Comment (jltha) [inline src/main/java/seedu/sherpass/command/AddCommand.java:80]: add newline

File: src/main/java/seedu/sherpass/command/AddCommand.java (hunk 1 above)
Comment (BradenTeo) [reply src/main/java/seedu/sherpass/command/AddCommand.java]: Sure, I can do this in another PR

Comment (BradenTeo) [reply src/main/java/seedu/sherpass/command/AddCommand.java]: Sure, will do.

**Review by yuzhongng — APPROVED** [review]

**Review by jltha — APPROVED** [review]

**Review by thedarie — APPROVED** [review]

**Pull request merged**